### PR TITLE
[RHACS] ROX-13807: Removing reference to telemetry from Helm chart documentation

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
3.74+

Issue:
https://issues.redhat.com/browse/ROX-13807

Link to docs preview:
- https://62177--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/install-central-other.html#central-services-public-configuration-file-central_install-central-other
- https://62177--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/install-central-ocp.html#central-services-public-configuration-file-central_install-central-ocp

SME review:
- [x] SME has approved this change.

Additional information:
Merge into `rhacs-docs` and cherry-pick into:
- `rhacs-docs-4.1`
- `rhacs-docs-4.0`
- `rhacs-docs-3.74`

### CODE REVIEW

File: modules/central-services-public-config.adoc [hunk 1]
```diff
@@ -96,8 +96,9 @@ include::snippets/technology-preview.adoc[]
 |`Central.declarativeConfiguration.mounts.secrets`
 | Mounts secrets used for declarative configurations.
 
-| `central.disableTelemetry`
-| Use `true` to disable online telemetry data collection.
+//ROX-13807
+//| `central.disableTelemetry`
+//| Use `true` to disable online telemetry data collection.
 //TODO: Add link to telemetry
```
Comment (stehessel) [inline modules/central-services-public-config.adoc:102]:
```suggestion
```

Why not delete those lines instead of commenting them out?

Comment (agantony) [reply modules/central-services-public-config.adoc]: Sure, we can do that.

**Comment by agantony** [comment]
Hi @kcarmichael08 Please can you do a final review and merge this PR, thx!

**Review by stehessel — APPROVED** [review]

**Comment by kcarmichael08** [comment]
LGTM

/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**